src/Cardano/Wallet/Api/Types.hs

```diff
@@ -28,31 +28,21 @@
 module Cardano.Wallet.Api.Types
     (
     -- * API Types
-      Address (..)
-    , Wallet (..)
+      ApiAddress (..)
+    , ApiWallet (..)
     , WalletBalance (..)
     , WalletPostData (..)
     , WalletPutData (..)
     , WalletPutPassphraseData (..)
 
-    -- * Re-Export From Primitives
-    , AddressState (..)
-    , PoolId (..)
-    , WalletDelegation (..)
-    , WalletId (..)
-    , WalletName (..)
-    , WalletPassphraseInfo (..)
-    , WalletState (..)
-    , AddressPoolGap
-    , Passphrase(..)
```

Comment:
removed the re-export. Consumer modules just have to import the `Cardano.Wallet.Primitive.Types` :+1: